### PR TITLE
blockdev_inc_backup_no_bitmap: Do incremental live backup without bitmap

### PR DESCRIPTION
**Pull request by aliang123**
Do incremental live backup without bitmap

Signed-off-by: Aihua Liang <aliang@redhat.com>
id:1983923

### CODE REVIEW

**Comment by aliang123** [comment]
(1/1) Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.x86_64.io-github-autotest-qemu.blockdev_inc_backup_no_bitmap.q35: PASS (44.49 s)

**Comment by aliang123** [comment]
@zhencliu @ngu-niny, please help to review, thanks.

File: qemu/tests/blockdev_inc_backup_no_bitmap.py [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    def _init_arguments_by_params(self, tag):
+        image_params = self.params.object_params(tag)
+        image_chain = image_params.objects("image_backup_chain")
+        self.source_images.append("drive_%s" % tag)
+        self.full_backups.append("drive_%s" % image_chain[0])
+        self.inc_backups.append("drive_%s" % image_chain[1])
+        self.bitmaps.append("bitmap_%s" % tag)
+
+    def do_full_backup(self):
+        extra_options = {"sync": "full", "auto_disable_bitmap": False}
```
Comment (zhencliu) [inline qemu/tests/blockdev_inc_backup_no_bitmap.py:29]: auto_disable_bitmap is not used, we can remove it.

File: qemu/tests/blockdev_inc_backup_no_bitmap.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            self.full_backups,
+            self.bitmaps,
+            **extra_options)
+
+    def generate_inc_files(self):
+        return list(map(self.generate_data_file, self.src_img_tags))
+
+    def do_incremental_backup(self):
+        extra_options = {"sync": self.inc_sync_mode,
+                         "auto_disable_bitmap": False}
```
Comment (zhencliu) [inline qemu/tests/blockdev_inc_backup_no_bitmap.py:42]: the same as above

File: qemu/tests/blockdev_inc_backup_no_bitmap.py (hunk 1 above)
Comment (aliang123) [reply qemu/tests/blockdev_inc_backup_no_bitmap.py]: have update, thanks zhenchao.

File: qemu/tests/blockdev_inc_backup_no_bitmap.py (hunk 2 above)
Comment (aliang123) [reply qemu/tests/blockdev_inc_backup_no_bitmap.py]: have updated, thanks zhenchao.

**Comment by ngu-niny** [comment]
1-Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.9.0.0.ppc64le.io-github-autotest-qemu.blockdev_inc_backup_no_bitmap Finshed 2021-07-28 02:14:37 2021-07-28 02:17:08 PASS 151.342349

Acked-by: Nini Gu <ngu@redhat.com>

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**